pallets/creditcoin/src/migrations/v8.rs

```diff
@@ -27,10 +26,38 @@ impl<Runtime> Migration<Runtime> {
 	}
 }
 
-impl<T: CreditcoinConfig> Migrate for Migration<T> {
-	fn pre_upgrade(&self) {
+impl<T: Config> Migrate for Migration<T> {
+	fn pre_upgrade(&self) -> Vec<u8> {
 		let count = Authorities::<T>::iter().count();
+
 		assert!(count != 0, "Authorities not found during migration");
```

Comment:
Question: isn't this going to fail on testnet? IDK if we have authorities configured there?

Reply:
IMO, we should. How can we do testing without Authorities?

Reply:
This assert is passing against testnet - https://github.com/gluwa/creditcoin/actions/runs/4147597388/jobs/7175167790. Marking as resolved.